test/adapters/flexibleLeverageStrategyAdapter.spec.ts

```diff
@@ -1542,11 +1629,13 @@ describe("FlexibleLeverageStrategyAdapter", () => {
   describe("#iterateRebalance", async () => {
```

Comment:
Can we write a test with 2 exchanges, we call iterateRebalance on both exchanges immediately after one another, where the global and exchange timestamps are updated correctly